src/App/Components/Comment/useEdit.js

```diff
@@ -0,0 +1,16 @@
+import { useState } from 'react';
```

Comment:
this opens up a textarea with the content of the recent reply or comment that a user wants to update